### ISSUE TITLE
UI is too small on HiDPI displays / UI doesn't respect display scaling

### ISSUE THREAD

**Issue by omeryagmurlu** [jellyfin/jellyfin-desktop#119]
**Is your feature request related to a problem? Please describe.**
On HiDPI displays, the UI doesn't scale up to the scaling set by the OS. I'm on Windows with 125% scaling, and the UI is small compared to the web client on a browser. One can use two-finger gestures on a trackpad to temporarily increase the zoom on the client, but this doesn't persist onto subsequent restarts of the application.

**Describe the solution you'd like**
The application gets the currently used scaling from Windows/GNOME/KDE/etc. and scales up accordingly.

**Describe alternatives you've considered**
At the very least, the application should allow changing the zoom used manually.

**Additional context**
I use custom CSS, but it's unrelated. I'm also using Windows 11, but the issue is also present on Windows 10. I didn't test on other OSes. Don't mind the different thumbnails on the web client on the screenshots, I'm running it built from a custom branch right now.

**Screenshots**
web client with correct scaling:
![web](https://user-images.githubusercontent.com/8201544/126913357-b281f478-e628-4ea8-8471-ae23d8e47dcd.png)

desktop client by default:
![image](https://user-images.githubusercontent.com/8201544/126913417-da3a7aca-88c7-40c8-8d0e-037726be2bd0.png)

desktop client after zooming in using 2-finger gestures (temporary) (how it should be):
![image](https://user-images.githubusercontent.com/8201544/126913462-e9901a78-c2e3-48bd-a10d-e42033377726.png)

**Comment by TheFeelTrain** [comment]
This isn't documented anywhere for Jellyfin yet but it retains the launch commands from Plex Media Player. So create a shortcut for the exe and add the `--scale-factor` argument. Hopefully they add a setting for it within the app though.

I use Linux so mine looks like this `jellyfinmediaplayer --scale-factor 1.25`

**Comment by zoodayz** [comment]
Yes I agree this would be a very helpful feature.  and Thank you TheFellTrain for this tip.   Quick question though does this just scale the Ui ie: text or is it the whole picture too?  thank you.

**Comment by jluttine** [comment]
Can this be set in `~/.local/share/jellyfinmediaplayer/jellyfinmediaplayer.conf` somehow?
